lightning/src/ln/channel.rs

```diff
@@ -686,6 +686,43 @@ impl UnfundedChannelContext {
 	}
 }
 
+/// Info about a transaction and its confirmation status, used mainly for funding transactions.
+pub(super) struct TransactionConfirmation {
```

Comment:
`ConfirmedTransaction` is probably a better way of naming this given the transaction itself is the primary data and the confirmation data is secondary.

Reply:
Though, I guess if it becomes unconfirmed then the name doesn't apply... hmmm

Reply:
How about `ConfirmingTransaction`? Or just leave `TransactionConfirmation`.
`TransactionAndConfirmationStatus` is too much.

Reply:
we use `FundingCandidate`

Reply:
Perhaps `FundingTransaction` given we use this for funded channels, too. It's essentially encapsulating information about that transaction.